src/libmongoc/src/mongoc/mongoc-log-private.h

```diff
@@ -34,4 +34,14 @@ mongoc_log_trace_bytes (const char *domain, const uint8_t *_b, size_t _l);
 void
 mongoc_log_trace_iovec (const char *domain, const mongoc_iovec_t *_iov, size_t _iovcnt);
 
+BSON_BEGIN_DECLS
+#define STOP_LOGGING_CHECK                                          \
```

Comment:
This macro is only used in `mongoc_log_trace_bytes()` and `mongoc_log_trace_iovec()`, not `mongoc_log()` as the JIRA ticket recommends. This is because the current check in `mongoc_log()` needs information on whether the log level is `TRACE`, whereas this information is implicit in the two functions that this macro is used in.